src/main/java/com/switchfully/switchfullylmsbackend/controller/ClassGroupController.java

```diff
@@ -16,7 +17,7 @@ public ClassGroupController(ClassGroupService classgroupService) {
     }
 
     @ResponseStatus(HttpStatus.CREATED)
-    @PostMapping(consumes = "application/json", produces = "application/json")
+    @PostMapping(path="/create-classgroup", consumes = "application/json", produces = "application/json")
```

Comment:
path /create-classgroup seems unneccessary when there is an open post request slot on /classgroup